Doc/library/textwrap.rst

```diff
@@ -17,31 +17,41 @@ If you're just wrapping or filling one or two text strings, the convenience
 functions should be good enough; otherwise, you should use an instance of
 :class:`TextWrapper` for efficiency.
 
-.. function:: wrap(text, width=70, **kwargs)
+.. function:: wrap(text, width=70, *, initial_indent="", \
+                   subsequent_indent="", expand_tabs=True, \
+                   replace_whitespace=True, fix_sentence_endings=False, \
+                   break_long_words=True, drop_whitespace=True, \
+                   break_on_hyphens=True, tabsize=8, *, max_lines=None, \
+                   **kwargs)
 
    Wraps the single paragraph in *text* (a string) so every line is at most
    *width* characters long.  Returns a list of output lines, without final
-   newlines.
+   newlines.  *width* defaults to ``70``.
```

Comment:
Why duplicate info that is already visible in the signature?

Reply:
This is just carried over from the current doc, and not relevant to the bpo, so I left it:

https://docs.python.org/3/library/textwrap.html#textwrap.wrap

Reply:
Oh, my bad, in the old documentation, that sentence was in the docs for `textwrap.wrap`, and for some reason I though it'd be a good idea to bring it down here so that they match, but that's definitely not right! If anything, it'd be better to remove the unnecessary repetition from `textwrap.wrap`, but again, that's out of the scope of this bpo, and we can make a new little bpo for that problem after this one lands.

Reply:
Either way, I'm working on reverting all changes to things other than the function signatures right now, because I think that's all we really want to do here.